src/server_manager/web_app/app.ts

```diff
@@ -53,6 +53,15 @@ function isManualServer(testServer: server.Server): testServer is server.ManualS
   return !!(testServer as server.ManualServer).forget;
 }
 
+async function isAccessKeyPortEditable(server: server.Server): Promise<boolean> {
+  try {
+    await server.setPortForNewAccessKeys(server.getPortForNewAccessKeys());
```

Comment:
As discussed offline, let's use a negative value as a sentinel to guarantee idempotence.

Reply:
Done